test/test_sqlalchemy_bigquery.py

```diff
@@ -345,7 +346,10 @@ def test_session_query(session, table, session_using_test_dataset, table_using_t
                 table.c.string,
                 col_concat,
                 func.avg(table.c.integer),
-                func.sum(case([(table.c.boolean == True, 1)], else_=0))
```

Comment:
FYI: I was getting a test failure for "is True". Changing to compare with equality and a sqlalchemy literal fixed the test case.

Reply:
Thanks a lot!